### ISSUE TITLE
Using Farsi or English for the documentations?

### ISSUE THREAD

**Issue by HamidMolareza** [HamidMolareza/Quera#52]
According to the audience of Quera site, it is possible that most people are not comfortable with English language. That's why it might be better to use Persian language.

**Comment by HamidMolareza** [comment]
@ARiYaNSEp0-0

I’d be happy to hear your thoughts on this.

**Comment by ARiYaNSEp0-0** [comment]
It's very good idea. and we keep them both or just one ? and what about organizing English and Persian repo and updating the content ?

**Comment by HamidMolareza** [comment]
We can have both languages side by side, but one should be the primary and default.  
The primary language file will be named `README.md`, which GitHub loads by default.  
The secondary language file can be named something like `README.en.md`.
Finally, can reference in readme files like the below:

### Languages
🇮🇷 [Persian](README.md) | 🌐 [English](README.en.md)

We should consider that our primary audience is Persian, and this repository is not useful for others.

**Comment by FatemehMOOHAMADI** [comment]
Hi Mr.Molareza, I'm Fatemeh Mohammadi. I'm fairly competent in English and I know Git/GitHub and I intend to participate in open source projects to get more real world practice, moreover collaboration with others sounds very interesting. 
I would like to participate in your projects if you will and join you in this journey.

**Comment by HamidMolareza** [comment]
Hi, Thank you for reaching out and expressing interest in collaborating.
You’re welcome to contribute to any project you like.
- [HamidMolareza](https://github.com/HamidMolareza/)
- [Payadel](https://github.com/Payadel/)

Contributions can be in the form of code, documentation, or even sharing ideas and feedback about the project.  
If you have any suggestions or feedback, please share them in the project's issue tracker. After that, if you'd like, you can create a PR.  
If you have any questions about how to contribute, feel free to ask me.

**Comment by FatemehMOOHAMADI** [comment]
I need to know where to begin? And what is in priority right now that needs
to be done?

**Comment by HamidMolareza** [comment]
@FatemehMOOHAMADI 
What languages and frameworks do you know, are familiar with, or want to learn? (C#, javascript, backend, front....?)

**Comment by FatemehMOOHAMADI** [comment]
I would like to help with translation works to be honest, and I've picked up backend as my learning path and regarding the courses I've taken I am about to complete CS50 Harvard computer science course, my programming language is python(Sabz learn academy) but I haven't learned any frameworks ,yet. 
I'm eager to learn and apply them; thus for starter I can do translation work related.

**Comment by FatemehMOOHAMADI** [comment]
Yes, please tell more detail about how you would like it to be done

**Comment by HamidMolareza** [comment]
@FatemehMOOHAMADI 
I create and invite you as collaborator to [this repository](https://github.com/HamidMolareza/readme-pe).

**Comment by FatemehMOOHAMADI** [comment]
Thank you
